source/features/show-open-prs-of-forks.tsx

```diff
@@ -67,7 +67,7 @@ async function initHeadHint(): Promise<void | false> {
 		return false;
 	}
 
-	select('.fork-flag .text')!.append(
+	select('.fork-flag .text, .hide-full-screen span.text-small')!.append(
```

Comment:
Same as before I think:

```suggestion
	select<HTMLAnchorElement>(`[data-hovercard-type="repository"][href="/${getForkSourceRepo()}"]`)!.append(
```